test/TestExtensions/SampleCommandLineExtensions/SampleCommandLineExtensions.csproj

```diff
@@ -4,9 +4,8 @@
   
   <PropertyGroup>
     <TargetFramework>$(NETFXTargetFramework)</TargetFramework>
-  </PropertyGroup>
-  <PropertyGroup>
     <RootNamespace>SampleCommandLineExtensions</RootNamespace>
+    <Description>A sample commandline extensions for testing NuGet.exe command extensibility</Description>
```

Comment:
typo: A sample **of** **command-line**

Reply:
Don't think that reads super well. I went with
`A sample commandline extension for testing NuGet.exe command extensibility` 

Let me know what you think.

Reply:
I think it would be better if it got changed to ``A sample command-line extension for testing NuGet.exe command extensibility.``

Reply:
Still prefer the hypen as well.

Reply:
ok, fixed.